app/Http/Controllers/MemberProjectController.php

```diff
@@ -4,44 +4,61 @@
 
 use Illuminate\Http\Request;
 use App\Http\Requests\StoreCreateMemberProject;
+use App\Http\Requests\StoreEditMemberProject;
 use App\MemberProject;
 
 class MemberProjectController extends Controller
 {
-    public function listMemberProject()
+    public function index()
     {
-        $listMemberProject=MemberProject::all()->toArray();
+        $listMemberProject = MemberProject::all()->toArray();
         return $listMemberProject;
     }
-    public function deleteMemberProject($id)
+
+    public function create()
+    {
+    }
+
+    public function show($id)
```

Comment:
Pls remove this. I dont see anything in this function.

Reply:
i have remove